dallinger/frontend/templates/thanks.html

```diff
@@ -0,0 +1,59 @@
+<!doctype html>
+<html>
+    <head>
+        <title>Psychology Experiment - Thanks</title>
+        <link rel=stylesheet href="/static/css/bootstrap.min.css" type="text/css">
+        <link rel=stylesheet href="/static/css/style.css" type="text/css">
+        <script src="/static/lib/jquery-min.js" type="text/javascript"> </script>
+        <style>
+          body {
+              padding:0px;
+              margin: 0px;
+              background-color: white;
+              color: black;
+              font-weight: 300;
+              font-size: 13pt;
+          }
+
+          #adlogo {
+              float: right;
+              width: 140px;
+              padding: 2px;
+              border: none;
+          }
+
+          #container-not-an-ad {
+              position: absolute;
+              top: 0px; /* Header Height */
+              bottom: 0px; /* Footer Height */
+              left: 0px;
+              right: 0px;
+              padding: 100px;
+              padding-top: 5%;
+              border: 18px solid #f3f3f3;
+              background: white;
+          }
+        </style>
+    </head>
+    <body>
+        <div id="container-not-an-ad">
+      <div class="well">
+        <h1>Thanks for your participation</h1>
+        <hr>
+
+        {% if mode == 'debug' %}
+          <p>If this were a real HIT, you would push a button to finish at this stage.
+        {% else %}
+          <p>To complete the HIT, simply press the button below.</p>
```

Comment:
For sandbox/production modes, is there a reason we can't or don't want to submit the form automatically?

For debug mode, I think we still want to have a form, which submits to an endpoint that calls `dallinger.experiment_server.experiment_server._debug_notify` the way that the `/ad_address` endpoint used to. That way we can also debug things that happen in response to notifications when working locally.